src/compas_fab/robots/robot.py

```diff
@@ -1430,13 +1615,40 @@ def plan_motion(self, goal_constraints, start_configuration=None,
         return trajectory
 
     def transformed_frames(self, configuration, group=None):
-        """Returns the robot's transformed frames."""
+        """Returns the robot's transformed frames.
+
+        Parameters
+        ----------
+        configuration : :class:`Configuration`
+            Configuration to compute transformed frames for.
+        group : str, optional
+            The planning group used for the calculation. Defaults to the robot's
+            main planning group.
+
+        Returns
+        -------
+        list of :class:`compas.geometry.Frame`
+        """
         joint_names = self.get_configurable_joint_names(group)
         joint_state = dict(zip(joint_names, configuration.values))
         return self.model.transformed_frames(joint_state)
 
     def transformed_axes(self, configuration, group=None):
-        """Returns the robot's transformed axes."""
+        """Returns the robot's transformed axes.
+
+        Parameters
+        ----------
+        configuration : :class:`Configuration`
+            Instance of the configuration (joint state) to use to compute
```

Comment:
Since this is complementary to `def transformed_frames` I would use the exact same sentence: `Configuration to compute transformed axes for.`

Reply:
71d5122